native/ios/WatermelonDB.xcodeproj/project.pbxproj

```diff
@@ -353,7 +353,7 @@
 					"$(SRCROOT)/../../../react-native/React/**",
 					"$(SRCROOT)/../../node_modules/react-native/Libraries/**",
 					"$(SRCROOT)/../../../../node_modules/react-native/Libraries/**",
-					"$(SRCROOT)/../../../../../ios/Pods/Headers/Public/**",
+					"$(SRCROOT)/../../../../../ios/Pods/Headers/Public/React-Core/**",
```

Comment:
could you explain your rationale behind this change?

Reply:
This fixes an issue where Watermelon fails to build on projects using Flipper, as there appears to be a conflict with the version of Folly that is bundled with it in react native 0.62.  As Watermelon only relies on React-Core, it doesn't need to search the entire pods folder recursively.

Reply:
See the discussion in this issue for more info:
https://github.com/Nozbe/WatermelonDB/issues/655

Reply:
@radex Do you have any comments on this?

Reply:
This PR could unblock my team, any update @radex ?

Reply:
@district-michael And doesn't this change break Watermelon for RN < 0.62?

Reply:
@radex I just tested in our project on 0.61 prior to our upgrade and it works fine.  Don't have any code prior to that version to test on.

Reply:
OK, cool! I'll test this on my project, and hopefully merge sometime soon :)

Reply:
I had the same issue. Just spent a whole day looking into it. Any plans to merge it?
FYI:
```js
"@nozbe/watermelondb": "^0.16.0",
"react": "16.11.0",
"react-native": "0.62.2",
```